pages/create-video/remotion/MyComp/Main.tsx

```diff
@@ -0,0 +1,104 @@
+// import { z } from "zod";
+import {
+  Sequence,
+  // spring,
+  // useCurrentFrame,
+} from "remotion";
+// import { CompositionProps } from "../../types/constants";
+import React from "react";
+// import { TextFade } from "./TextFade";
+// import { IssueTags } from "../../../../components/IssueTags/IssueTags";
+
+const tag: React.CSSProperties = {
+  width: "140pt",
+  height: "40pt",
+  padding: "24pt 14pt",
+  borderRadius: "32pt",
+  backgroundColor: "#003C55",
+  fontFamily: "proxima_nova",
+  fontSize: "2.25rem",
+  color: "white",
+  display: "flex",
+  justifyContent: "center",
+  alignItems: "center",
+};
+
+const bigTag: React.CSSProperties = {
+  width: "48rem", // 270pt
+  height: "12rem", // 68pt
+  padding: "1.25rem 1.875rem", // 20pt 30pt
+  borderRadius: "3.75rem",
+  border: "4px solid #FFB45C",
+  backgroundColor: "rgba(255, 180, 92, 0.1)", // 10% opacity
+  fontFamily: "commuter_sans",
+  fontSize: "4rem",
+  color: "white",
+  display: "flex",
+  justifyContent: "center",
+  alignItems: "center",
+  boxShadow: "0px 0px 5px rgba(0, 0, 0, 0.3)",
+};
+
+export const Main = () => {
+  return (
+    <Sequence>
+      <div
+        style={{
+          display: "flex",
+          flexDirection: "column",
+          alignItems: "left",
+          justifyContent: "top",
+          padding: "8rem 4rem", // 120pt 40pt
+          gap: "4rem",
+        }}
```

Comment:
Ideally we eliminate these inline styles in favor of using a SCSS module to keep the markup clean

Reply:
Yup! Inlining for now for dev speed + iteration. Will move to module later.